docs/README-EDITING.md

```diff
@@ -66,77 +51,120 @@ sitemap:
 ---
 ```
 
-### Subdirectories
+## Editing the docs
 
-I recommend to keep files flat without subdirectories. 
-If subdirectory being to add, it is required to update sidebars creating logic.
+When editing the docs content, follow the guidelines below.
+ 
+### Editing the `next` docs version
 
-### Prefer .md, avoiding .vue
+Files to be included in the docs' next version live in the `next` directory.
 
-The `*.vue` allows us to use vue pages and components. It is undesirable because it introduces additional complexity. 
-For that purposes `.vue` suffix is disabled.
+The `next` version of the docs is available only locally and on the staging server.
 
-Keeping all content as just `.md` file makes them more manageable.
+To display the `next` version in a browser:
+1. Start a [local Handsontable docs server](./README.md#getting-started-with-handsontable-docs).
+2. In your browser, go to http://localhost:8080/docs/next/.
 
-## Editing documentation content
+### Editing the `latest` docs version
 
-### Live demo:
-It is able to run code snippets to show result of the code:
+A [`<semver.version>` directory](./README.md#handsontable-docs-directory-structure) with the largest version number gets tagged as the `latest` version of the docs.
 
-```
-    ::: example #example1
+To display the latest docs version in a browser:
+1. Start a [local Handsontable docs server](./README.md#getting-started-with-handsontable-docs).
+2. In your browser, go to http://localhost:8080/docs.
```

Comment:
Please add information, that it is required to update both the latest and next version while editing the latest version.